website/content/platform/development/developer-guidelines/architectural_considerations.md

```diff
@@ -97,12 +97,10 @@ The `Fetcher` class implementation is based on the [TET pattern](/platform/devel
 
 The OBBject class is a generic class in the OpenBB platform that represents a standardized object for handling and manipulating data fetched from various providers. It extends the `Tagged` class and uses Python's generics to allow flexibility in the type of results it can handle.
 
-The class includes several fields such as results, provider, warnings, chart, and extra, which respectively represent the fetched data, the data provider, any warnings generated during data fetching, an optional chart object for visualizing the data, and a dictionary for any additional information.
+The class includes several fields such as results, provider, warnings, and extra, which respectively represent the fetched data, the data provider, any warnings generated during data fetching, and a dictionary for any additional information.
 
 The class provides several methods for converting the fetched data into different formats, including `to_df` (or `to_dataframe`) for converting to a pandas DataFrame, `to_polars` for converting to a Polars DataFrame, `to_numpy` for converting to a numpy array, and `to_dict` for converting to a dictionary.
 
-The `to_chart` method allows for creating or updating a chart based on the fetched data, and the show method is used to display the chart.
-
 The class also includes a `__repr__` method for a human-readable representation of the object, and a `model_parametrized_name method` for returning the model name with its parameters.
```

Comment:
@hjoaquim, when I run, `data.model_parameterized_name()`, I get:

```
TypeError: OBBject.model_parametrized_name() missing 1 required positional argument: 'params'
```

Reply:
Indeed. Not sure what the intention was here tbh, but we can either remove it or change it to `schema()` instead. Wdyt?